assets/router/deployment.yaml

```diff
@@ -4,6 +4,7 @@ kind: Deployment
 apiVersion: apps/v1
 # name and namespace are set at runtime.
 spec:
+  progressDeadlineSeconds: 120
```

Comment:
This reminds me that we need to set the readiness probe to use `/healthz/ready`.  I'm nervous about setting the progressing deadline significantly lower than the default, especially if we start using `/healthz/ready`.  Are we confident that the initial sync will finish in time on large clusters?

Reply:
The default is 600 seconds, which seems too long. Do you agree? Would some e2e test warn us if we chose a value that's too short on average?

Reply:
> The default is 600 seconds, which seems too long. Do you agree?

That's what I'm wondering.

> Would some e2e test warn us if we chose a value that's too short on average?

Not if the E2E tests are not representative of production clusters.  Moreover, while we may be fine now, I intend to fix the readiness check to use `/healthz/ready`, which will cause the deployment not to be ready until the router has synched routes, which I could see taking more than 120 seconds on burdened clusters with many routes.

Reply:
Progress deadline is usually measured in 10-20m.

Reply:
This is way too low.

Reply:
We're also not setting readiness endpoints correctly, fixing that and changing back to 10m in a followup.

Reply:
Nevermind, fixed here instead